### PR TITLE
DDF-1825 Added documentation for exporting from Admin UI

### PR DESCRIPTION
**Pull request by andrewkfiedler**
- Added three pictures that show the steps to export.
 - Added some steps that also explain how to export.

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/496" target="_blank">ReviewNinja

**Comment by andrewkfiedler** [comment]
@garrettfreibott , @lessarderic , @figliold , @ricklarsen 

Should I add in information on what could go wrong during an export?

**Comment by andrewkfiedler** [comment]
exporting docs

Here is what it currently looks like.

**Comment by ricklarsen** [comment]
I think a "Troubleshooting" section might be a good idea if there are likely to be common things that might pop up that can be easily addressed.

File: platform/admin/docs/src/main/resources/ManContents.adoc [hunk 1]
```diff
@@ -237,3 +237,17 @@ A system restart is required for the changes to `users.properties` to take effec
...
+
+=== Exporting Configurations
+
+You can export the current system configurations using the Admin UI.  This is useful for migrating from one running instance to another.
+
+To do so, follow these instructions:
+
+. Select the `System` tab (next to the Applications tab) image:exporting_configuration_step1.png[Exporting Step 1,${image.width}]
+. Click the `Export` button image:exporting_configuration_step2.png[Exporting Step 2,${image.width}]
+. Fill out the form, specifying where you want to export the configurations to image:exporting_configuration_step3.png[Exporting Step 3,${image.width}]
```
Comment (lessarderic) [inline platform/admin/docs/src/main/resources/ManContents.adoc]: Do we need the last `to` here?

**Comment by andrewkfiedler** [comment]
exporting docs troubleshooting

**Comment by lessarderic** [comment]
Question for @ricklarsen. In general, do we use screenshots in our documents for things like this? They are useful, but they tend to make the documentation much larger and more difficult to maintain, so I was curious to know what our take on this was.

File: platform/admin/docs/src/main/resources/ManContents.adoc [hunk 2]
```diff
@@ -237,3 +237,28 @@ A system restart is required for the changes to `users.properties` to take effec
...
+
+=== Exporting Configurations
+
+You can export the current system configurations using the Admin UI.  This is useful for migrating from one running instance to another.
+
+To do so, follow these instructions:
+
+. Select the `System` tab (next to the Applications tab) image:exporting_configuration_step1.png[Exporting Step 1,${image.width}]
+. Click the `Export` button image:exporting_configuration_step2.png[Exporting Step 2,${image.width}]
+. Fill out the form, specifying where you want to export the configurations image:exporting_configuration_step3.png[Exporting Step 3,${image.width}]
```
Comment (lessarderic) [inline platform/admin/docs/src/main/resources/ManContents.adoc]: Do we want to mention that the path points to a directory on the server, not on the user's local machine?

Comment (andrewkfiedler) [inline platform/admin/docs/src/main/resources/ManContents.adoc]: I'm not sure how much we want to duplicate things.  The tooltip for the field already mentions this, but I'll add it here as well.

File: platform/admin/docs/src/main/resources/ManContents.adoc (hunk 1 above)
Comment (ricklarsen) [inline platform/admin/docs/src/main/resources/ManContents.adoc]: I'd maybe rewrite to "...specifying the destination for the export."

**Comment by ricklarsen** [comment]
I agree with you @lessarderic that we should make a firm decision regarding screenshots. Personally, I could go either way with including them or not, but it should be part of any UI change to update any snapshots as needed.

**Comment by lessarderic** [comment]
@andrewkfiedler, @ricklarsen, I just realized that the export documentation was added to the platform `mancontents.adoc` file. Is that what we want to do moving forward? I though that most of the documentation was going in the doc files under distribution?

**Comment by ricklarsen** [comment]
@lessarderic @andrewkfiedler I think that's a good point. I have some configuration export details in the managing DDF section already. I think it's good to have under distribution/docs because it's a system-wide functionality, and the platform/admin/docs should just details for configuring/managing the admin ui itself.

That said, I am pushing up some major changes to distribution/docs today, so it might be more practical to put it here and I'll move it to distribution when I refactor platform/admin

**Pull request merged**